buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -111,11 +110,18 @@ Requires: boost-python3-devel
 %endif
 
 %if 0%{?el7}
-Requires: boost-python36-devel
-Requires: python36-devel
-Requires: python36-h5py
+Requires: python36-setuptools
+Requires: python36-qt5-devel
 Requires: python36-numpy
+Requires: python36-scipy
+Requires: python36-sphinx
+Requires: python36-dateutil
+Requires: python36-h5py
 Requires: python36-PyYAML
+Requires: python36-mock
+Requires: python36-psutil
+Requires: python36-requests
+Requires: boost-python36-devel
```

Comment:
Can we add `boost169-python2-devel` and `boost169-python3-devel`  to this list?

Reply:
:+1: